eo-maven-plugin/src/test/java/org/eolang/maven/FpUpdateFromCacheTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2016-2025 Objectionary.com
+ * SPDX-License-Identifier: MIT
+ */
+package org.eolang.maven;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.cactoos.Text;
+import org.cactoos.text.TextOf;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * Test case for {@link FpUpdateFromCache}.
+ * @since 0.57
+ */
+final class FpUpdateFromCacheTest {
+
+    @Test
+    void appliesFromCache(@TempDir final Path tmp) throws IOException {
```

Comment:
@volodya-lombrozo it's better to use `@Mktmp` so the result is saved in temporary directory inside `target`